.github/workflows/codebuild-ci.yml

```diff
@@ -16,14 +16,31 @@ jobs:
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          role-to-assume: ${{ secrets.CI_AWS_ROLE_ARN }}
+          role-to-assume: ${{ secrets.CI_MAIN_TESTING_ACCOUNT_ROLE_ARN }}
+          role-duration-seconds: 7200
+          aws-region: us-west-2
+      - name: Setup .NET Core 6.0
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 6.0.x
+      - name: Invoke Load Balancer Lambda
+        id: lambda
+        shell: pwsh
+        run: |
+          aws lambda invoke response.json --function-name "${{ secrets.CI_TESTING_LOAD_BALANCER_LAMBDA_NAME }}" --cli-binary-format raw-in-base64-out --payload '{\"Roles\": \"${{ secrets.CI_TEST_RUNNER_ACCOUNT_ROLES }}\", \"ProjectName\": \"${{ secrets.CI_TESTING_CODE_BUILD_PROJECT_NAME }}\", \"Branch\": \"${{ github.sha }}\"}'
```

Comment:
The AWS CLI is natively available on the runner image https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md.
I am using AWS CLI primarily because I didn't need to install additional tooling (AWS Tools for PowerShell). Also, during testing, I did test out AWS Tools for PowerShell and it piping the output of the command to later steps wasn't straightforward so I stuck with AWS CLI.

Reply:
Cool, didn't realize it was provided by GitHub.